drivers/SmartThings/zigbee-lock/src/init.lua

```diff
@@ -34,8 +34,12 @@ local UserStatusEnum            = LockCluster.types.DrlkUserStatus
 local UserTypeEnum              = LockCluster.types.DrlkUserType
 local ProgrammingEventCodeEnum  = LockCluster.types.ProgramEventCode
 
+local socket = require "socket"
 local lock_utils = require "lock_utils"
 
+local DELAY_LOCK_EVENT = "_delay_lock_event"
+local MAX_DELAY = 10
```

Comment:
So if there is a lock that doesn't send the LockOperationEvent, we will be delaying lock state capability events for 10 seconds. Is that right? If so, why 10, and should it instead be like 1-3 seconds?

Reply:
This _should_ be written in a way such that if a lock _never_ sends a LockOperationEvent, we just send the event immediately.

Reply:
i.e. only a device that sends _both_ can have its messages delayed, and then by a max of 10 seconds